.github/workflows/md-external-links-check.yml

```diff
@@ -0,0 +1,15 @@
+name: Check external links
+on:
+  workflow_dispatch:
+  schedule:
```

Comment:
Are us sure we will see the result of this task?
I understand you are not doing it in PR to not break a PR on something which does not depend of its content but I am afraid that we never see the status of the PR.

Maybe we should also provide a more visible result

Reply:
> we can:
1. open an issue on failure using something like https://github.com/JasonEtco/create-an-issue
2. send an email https://github.com/marketplace/actions/send-email

I would go for 1. WDYT?

Reply:
I would go for 1 as well

Reply:
I would go for one (but you do not need a github action for that, a simple script would do the job)

Reply:
@kpanot I have added a simple script that will create the issue. However, this will always create a new issue, it won't check if an issue on this topic is already open.
If we want this, I would use the Github action (which provides this option).

Reply:
Why not doing something like this:
- create an issue with a specific title (if it does not exist another opened issue with the same title)
- Replace the body of the issue with the specific title (new issue or previous one) with the list of the broken links as check box
?

Reply:
> * Replace the body of the issue with the specific title (new issue or previous one) with the list of the broken links as check box

This can be cumbersome because `github-action-markdown-link-check` does not provide any output.